app/data/menuitems.js

```diff
@@ -31,7 +31,7 @@ export default [
     links: [
       {
         id: 6.1,
-        url: 'http://2017.jsheroes.io/',
```

Comment:
I would put /2018 first then /2017, to keep the reverse order on the long run

Reply:
Done